src/permission.ts

```diff
@@ -2,24 +2,22 @@ import { autoinject } from 'aurelia-dependency-injection';
 import { SessionService, SessionPort } from './session';
 
 export type PermissionType = 
-    'ar.stage'          //Geolocation
-    | 'ar.camera'       //Camera
-    | 'ar.3dmesh';      //3D Structural mesh
-
-export const PermissionNames = {
-        'ar.stage': 'Location',
-        'ar.camera': 'Camera',
-        'ar.3dmesh': 'Structural mesh'
-    };
-
-const PermissionDescriptions = {
-        'ar.stage': 'your location', 
-        'ar.camera': 'your camera',
-        'ar.3dmesh': 'the structure of your surroundings'
-    };
-/**
- * 
- */
+    'location'          //Geolocation
+    | 'camera'          //Camera
+    | 'depthmesh';      //3D Structural mesh
```

Comment:
"depthmesh" seems a bit too similar to depth map to me. Here are some other alternatives: "world-mesh", "world-structure", "world-terrain", "terrain-mesh", "environment-mesh", "context-terrain" ? "3d-mesh" works too. What do you think?